.bazelrc

```diff
@@ -32,7 +32,6 @@ build:cache --experimental_remote_cache_compression
 build:remote-shared --remote_upload_local_results
 build:remote-shared --remote_timeout=600
 build:remote-shared --remote_download_minimal
-build:remote-shared --experimental_repo_remote_exec
```

Comment:
not sure if there was a reason we added this originally, maybe @siggisim remembers - but at a glance it seems OK to remove

Reply:
Just wanted to try it out